docs/setting-up-a-validator.md

```diff
@@ -149,6 +149,29 @@ ExecStart=/usr/bin/gbt orchestrator \
 
 For the Geth node, if you are going to run a geth full node delete lines 11-15 and uncomment lines 17-21
 
+## Add Skip Invariants to Gravity Command
```

Comment:
invariant check is no longer required here.

Reply:
removed